src/test/java/com/alkemy/ong/mapper/NewsMapperTest.java

```diff
@@ -0,0 +1,165 @@
+package com.alkemy.ong.mapper;
+
+import com.alkemy.ong.dto.NewsDto;
+import com.alkemy.ong.entity.CategoryEntity;
+import com.alkemy.ong.entity.NewsEntity;
+import com.alkemy.ong.mapper.NewsMapper;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.mapstruct.factory.Mappers;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class NewsMapperTest {
+
+    private static NewsMapper mapper;
+
+    @BeforeAll
+    public static void setUp() {
+        mapper = Mappers.getMapper(NewsMapper.class);
+
+    }
+    @Test
+    void newsEntityToNewsDto() {
+        CategoryEntity category = new CategoryEntity();
+        category.setId("1234");
+        category.setImage("Imagen category");
+        category.setName("nombre category");
+        category.setSoftDelete(false);
+        category.setDescription("Description");
+        category.setTimestamps(LocalDateTime.now());
+        NewsEntity news = new NewsEntity();
+        news.setId("123e4567-e89b-12d3-a456-426614174000");
+        news.setName("Novedad");
+        news.setContent("Contenido");
+        news.setImage("Imagen");
+        news.setTimestamps(LocalDateTime.now());
+        news.setCategoryId(category);
+        news.setSoftDelete(false);
+        NewsDto newsDto = mapper.newsEntityToNewsDto(news);
+        assertAll(
+                () -> {
+                    assertEquals(newsDto.getId(), news.getId());
+                    assertEquals(newsDto.getName(), news.getName());
+                    assertEquals(newsDto.getContent(), news.getContent());
+                    assertEquals(newsDto.getImage(), news.getImage());
+                    assertEquals(newsDto.getTimestamps(), news.getTimestamps());
+                    assertEquals(newsDto.getCategoryId(), news.getCategoryId());
+                    assertEquals(newsDto.getSoftDelete(), news.getSoftDelete());
+                }
+        );
+    }
+    @Test
+    public void newsEntityToNewsDtoSimpleNullTest(){
+        NewsEntity news=null;
+        NewsDto dto=mapper.newsEntityToNewsDto(news);
```

Comment:
this can be written in one line like this: NewsDto dto=mapper.newsEntityToNewsDto(null);

Reply:
I think I need to initialize the NewsEntity to pass it to the "assertEquals(null,news);" May be i'm wrong and there is other way to do it.

Reply:
I understand